docs/dev/release.md

```diff
@@ -98,14 +98,20 @@ pointing to the directory where `operatorhub-operator` repository was cloned in
 
 For this PR the sources are copied from the `community-operators` folder instead of the one where the `mongodb-atlas-kubernetes` resides.
 
-Invoke with <version> like `1.0.0`:
+Invoke with <version> like `1.0.0` (never use the `v` prefix here, just the plain SEMVER version `x.y.z`):
```

Comment:
curiosity question: why no `v` prefix? (formatting?)

Reply:
I made that mistake myself in a PR for version 1.9.1 and that breaks the diff. The v prefix is used at some places (maybe git tags/branches) but not in others (like docker tags) so the code will put it for you where applicable, but if you added it, you break the process as it is designed now.